.vscode/settings.json

```diff
@@ -4,5 +4,74 @@
     ],
     "pylint.args": [
         "--rcfile=tox.ini"
-    ]
+    ],
+    "files.associations": {
```

Comment:
VS code keeps adding all these file associations on macos. I guess they are ok?

Reply:
They really seem right to me, unless the files truly have no `.h` extension. But maybe easier to check this in as vscode will keep adding them back?

Reply:
I guess this come from using the C++ standard headers, which doesn't have a file extension.